perun-core/src/main/java/cz/metacentrum/perun/core/impl/UsersManagerImpl.java

```diff
@@ -914,10 +914,6 @@ public void removeAllUserExtSources(PerunSession sess, User user) {
 				return jdbc.query("select " + userMappingSelectQuery + "  from users " +
```

Comment:
We don't plan to have a different DB backend for Perun, so you can probably remove whole ``if / else`` and simply use postgres SQL code.

Reply:
Fixed